unicon_backend/models/problem.py

```diff
@@ -62,7 +62,6 @@ class ProblemORM(CustomSQLModel, table=True):
         cascade_delete=True,
     )
     supporting_files: sa_orm.Mapped[list["FileORM"]] = Relationship(
-        passive_deletes=True,
```

Comment:
@seelengxd this was raising an warning because of it was contradicting the `viewonly` field that set to `true` below, wanted to confirm if we could delete this